markdown/usage/tutorials/nf_core_contributing_overview.md

```diff
@@ -42,20 +41,23 @@ In this tutorial we discuss the best-practice guidelines developed by the *nf-co
 ### What is *nf-core*
 
 *nf-core* is a community-led project to develop a set of best-practice pipelines built using Nextflow.
+
+The aim of *nf-core* is to develop with the community over GitHub and other communication channels to build a pipeline to solve a particular analysis task.
+We therefore encourage cooperation and adding features to existing pipelines.
+Before considering starting a new pipeline, have a look if a pipeline already exists performing a similar task and consider contributing to that one instead!
```

Comment:
Just to be safe, maybe also the usual line about introducing any new pipelines in #new-pipelines first

Reply:
yes, makes sense!